docs/guides/image-update.md

```diff
@@ -247,9 +247,12 @@ Create an `ImageUpdateAutomation` to tell Flux which Git repository to write ima
 flux create image update flux-system \
 --git-repo-ref=flux-system \
 --branch=main \
+--git-repo-path="./clusters/my-cluster" \
+--checkout-branch=main \
+--push-branch=main \
 --author-name=fluxcdbot \
 --author-email=fluxcdbot@users.noreply.github.com \
---commit-template="[ci skip] update image" \
+--commit-template="{{range .Updated.Images}}{{println .}}{{end}}" \
```

Comment:
This change removes the `[ci skip]` example, is that done on purpose, and if so, do we want to highlight this elsewhere?

Reply:
I don't think people should skip CI when pushing changes to the repo made with flux bootstrap.

Reply:
We mention skip CI in the docs here: https://toolkit.fluxcd.io/components/image/imageupdateautomations/#commit